pkg/cmd/attestation/api/client_test.go

```diff
@@ -173,14 +180,93 @@ func TestGetByDigest_Error(t *testing.T) {
 	require.Nil(t, attestations)
 }
 
+func TestFetchBundlesByURL(t *testing.T) {
+	httpClient := &mockHttpClient{}
+	client := LiveClient{
+		httpClient: httpClient,
+		logger:     io.NewTestHandler(),
+	}
+
+	att1 := makeTestAttestation()
```

Comment:
these could be empty as long as the BundleURL is not empty eh? Since we have an if statement on that distinction, might be nice to test it?